deploy/01_genesis.ts

```diff
@@ -222,42 +237,58 @@ const func: DeployFunction = async function (hre1: HardhatRuntimeEnvironment) {
             '610736565b805160208083015191908110156107f5577fffffffffffffffffff',
             'ffffffffffffffffffffffffffffffffffffffffffffff8160200360031b1b82',
             '1691505b50919050565b600060208083528351808285015260005b8181101561',
-            '08285785810183015185820160400152820161080c565b8181111561083a5760',
-            '00604083870101525b50601f017fffffffffffffffffffffffffffffffffffff',
-            'ffffffffffffffffffffffffffe016929092016040019392505050565b600060',
-            '20828403121561088057600080fd5b81517fffffffff00000000000000000000',
-            '000000000000000000000000000000000000811681146105b057600080fdfea1',
-            '64736f6c634300080d000a820a99a08171f54150d81230716b6ecf1bac659a25',
-            '716068feb2a3e51308454748ef47b5a00a1ac7967e2f480e1e07661365a12ad1',
-            '4a37df62a0303a62420f59f0dd408ca8',
+            '08285785810183015185820160400152820161080c565b506000604082860101',
+            '5260407fffffffffffffffffffffffffffffffffffffffffffffffffffffffff',
+            'ffffffe0601f8301168501019250505092915050565b60006020828403121561',
+            '087957600080fd5b81517fffffffff0000000000000000000000000000000000',
+            '0000000000000000000000811681146105b057600080fdfea164736f6c634300',
+            '0811000a820a9aa065edd5f689b14d3fcdc5808bc0e732055c70ccdd6f35430f',
+            '6d6bb896058c69b6a02a3128da3839c946051c2991ad92b4ed4ef7b885c1a3c9',
+            'efa6465bf438f49d1c',
+          ].join('')
+        );
+        // enable 0xC0FFEE78121f208475ABDd2cf0853a7afED64749 on localhost2
+        await hre.ethers.provider.sendTransaction(
+          [
+            '0x',
+            'f8a70180830186a0940c8af56f7650a6e3685188d212044338c21d3f7380b844',
+            'a07d7316000000000000000000000000c0ffee78121f208475abdd2cf0853a7a',
+            'fed6474900000000000000000000000000000000000000000000000000000000',
+            '00000001820a9aa0fdad2831e1ead5069c2c6e28e62afe0b1d563bc4242312bb',
+            '555b0388003f89d8a01aa43771970f8bc0bf421a3bbbe76f43df751844cad394',
+            'e80a8eb2d8b7cf1acc',
           ].join('')
         );
-        // enable deployer on localhost2
+        // enable 0xdf5295149F367b1FBFD595bdA578BAd22e59f504 on localhost2
```

Comment:
just for clarity update to enable 0xd078E391cBAEAa6C5785124a7207ff57d64604b7 as testnet deployer on localhost

Reply:
Added that in next commit